integrationTests/state/stateTrie_test.go

```diff
@@ -1025,6 +1032,152 @@ func TestAccountsDB_ExecALotOfBalanceTxOKorNOK(t *testing.T) {
 	adbPrintAccount(acntDest.(*state.Account), "Destination")
 }
 
+func BenchmarkCreateOneMillionAccountsWithMockDB(b *testing.B) {
+	nrOfAccounts := 1000000
+	nrTxs := 15000
+	txVal := 100
+	balance := nrTxs * txVal
+	persist := mock2.MockDB{}
+
+	adb, _ := createAccounts(b, nrOfAccounts, balance, persist)
+	var rtm runtime.MemStats
+	runtime.GC()
+
+	runtime.ReadMemStats(&rtm)
+	fmt.Printf("Mem before committing %v accounts with mockDB: go mem - %s, sys mem - %s \n",
+		nrOfAccounts,
+		core.ConvertBytes(rtm.Alloc),
+		core.ConvertBytes(rtm.Sys),
+	)
+
+	_, _ = adb.Commit()
+
+	runtime.ReadMemStats(&rtm)
+	fmt.Printf("Mem after committing %v accounts with mockDB: go mem - %s, sys mem - %s \n",
+		nrOfAccounts,
+		core.ConvertBytes(rtm.Alloc),
+		core.ConvertBytes(rtm.Sys),
+	)
+}
+
+func BenchmarkCreateOneMillionAccounts(b *testing.B) {
+	nrOfAccounts := 1000000
+	nrTxs := 15000
+	txVal := 100
+	balance := nrTxs * txVal
+	persist := mock2.NewCountingDB()
+
+	adb, addr := createAccounts(b, nrOfAccounts, balance, persist)
```

Comment:
This test is not relevant as it uses memory db. What does it test?

Reply:
This test benchmarks the time needed for executing 15000 txs on a partially collapsed trie, and on a totally collapsed one, and shows the total number of nodes in a trie and the number of nodes that changed after the 15000txs are executed. Will change the memory measurements so it will show the difference between a collapsed trie and the same trie after the exec of 15000txs.